exporter/prometheusexporter/collector.go

```diff
@@ -36,6 +37,12 @@ var (
 	separatorString = string([]byte{model.SeparatorByte})
 )
 
+var disableNormalizationGate = featuregate.Gate{
```

Comment:
this should be part of the exporter configuration, rather than a feature-gate.  Feature-gates are meant to be temporary.

Reply:
+1

Reply:
yeah misunderstood. will change accordingly if needed.